src/test/run-make/removing-code-and-incremental-lto/Makefile

```diff
@@ -0,0 +1,69 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# How to run this
+# $ ./x.py clean
+# $ ./x.py test --target thumbv7m-none-eabi src/test/run-make
+
+# The original target of interest was thumbv7em-none-eabi
```

Comment:
Oh, right, this probably needs an `# only-thumbv7em-none-eabi`.

(I was trying to figure out how/if to generalize beyond that target, and in the process I got distracted by filing #67018 )